### PR TITLE
feat: add GitHub release action

### PR DESCRIPTION
**Pull request by childish-sambino**

### CODE REVIEW

File: actions/release/ReleaseGitHub.ts [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+  getHeader(): string[] {
+    return [];
+  }
+
+  getFooter(version: string): string[] {
+    let footer: string[] = [];
+
+    if (this.context.repo.owner === "twilio") {
+      footer = [
+        `**[Docs](https://twilio.com/docs/libraries/reference/${this.context.repo.repo}/${version}/index.html)**`,
```
Comment (shwetha-manvinkurke) [inline actions/release/ReleaseGitHub.ts:160]: don't think this works for go

File: .gitignore [hunk 2]
```diff
@@ -11,7 +11,6 @@ __pycache__/
 env/
 build/
 develop-eggs/
-dist/
```
[the remark this answers is not in the document]
Comment (childish-sambino) [reply .gitignore:14]: It's discussed here as an alternative to checking in _node_modules_: https://docs.github.com/en/actions/creating-actions/creating-a-javascript-action#commit-tag-and-push-your-action-to-github

File: actions/release/ReleaseGitHub.ts (hunk 1 above)
Comment (childish-sambino) [reply actions/release/ReleaseGitHub.ts:160]: It doesn't, but it's also what's currently happening. Suggestions?

Comment (childish-sambino) [reply actions/release/ReleaseGitHub.ts]: Added a case for twilio-go.

**Review by shwetha-manvinkurke — APPROVED** [review]

**Pull request merged**